### ISSUE TITLE
Loss not converging  and can't load model

### ISSUE THREAD

**Issue by BuaaAlban** [SeanNaren/deepspeech.pytorch#416]
Hi sean,
I have met some problems

info:
dataset: librispeech   cuda9  python3.6

default parameters, on 4 gpus,  and the loss keep going on from the beginning of 100 to about 500 and even 800 and the WER =100, even with several epochs.

and then i tried to use the  Pre-trained models V1.2 

the same mistake with this problem https://github.com/SeanNaren/deepspeech.pytorch/issues/331#issue-343129952  occured, but I don't understand the answer, also in fact  I have make some changes as https://github.com/SeanNaren/deepspeech.pytorch/issues/387 to make it running

could you help me ?

**Comment by SeanNaren** [comment]
the pre-trained models are based of an older commit, you need to actually checkout 655cd58 and then the pre-trained models will run

**Comment by ishita1995** [comment]
Hi @SeanNaren , 
I'm facing similar problem as @BuaaAlban 
Info - 
librispeech cu10 python 3.6 

The training loss keeps constantly increasing from the very first epoch and WER is also very high even after running it for approx 15 epochs.
Please advise on how to tackle this problem

**Comment by BuaaAlban** [comment]
do not use mix-precision training may help


朱博康
北京航空航天大学
中法工程师学院
地址：北京市海淀区学院路37号
邮编：100191
电话：13051576002
邮箱：buaazbk@buaa.edu.cn


Zhu bokang
Beihang University
Ecole Centrele de Pekin
Adress:No.37 College Road, Haidian District, Beijing
Tel: 13051576002
E-mail: buaazbk@buaa.edu.cn
On 11/19/2019 20:12，ishita1995<notifications@github.com> wrote：

Hi @SeanNaren ,
I'm facing similar problem as @BuaaAlban
Info -
librispeech cu10 python 3.6

The training loss keeps constantly increasing from the very first epoch and WER is also very high even after running it for approx 15 epochs.
Please advise on how to tackle this problem

**Comment by SeanNaren** [comment]
I'm wondering if this is because of AMP/loss-scaling. Could you set `loss-scale` to 1, and the optimisation level to `O0`

**Comment by BuaaAlban** [comment]
> I'm wondering if this is because of AMP/loss-scaling. Could you set `loss-scale` to 1, and the optimisation level to `O0`

I am sorry that I have removed the project several months before,  I just remember that without mix-precision option helped the loss converge.
It seems you have committed a new version, at that time, there was just an option --mix-precison, I haven't change loss-scale

**Comment by ishita1995** [comment]
Thanks @SeanNaren It worked. The is trained and loss also fell.
